### PR TITLE
fix mixscape default setting of layer

### PR DESCRIPTION
**Pull request by xinyuejohn**
**PR Checklist**

-   [ ] Referenced issue is linked
-   [ ] If you've fixed a bug or added code that should be tested, add tests!
-   [ ] Documentation in `docs` is updated

**Description of changes**

**Technical details**

**Additional context**

### CODE REVIEW

File: pertpy/tools/_mixscape.py [hunk 1]
```diff
@@ -194,7 +194,9 @@ def mixscape(
         if layer is not None:
             X = adata_comp.layers[layer]
         else:
-            X = adata_comp.X
+            warning = "Using .layers[\"X_pert\"] as default!"
+            print(warning)
```
Comment (Zethson) [inline pertpy/tools/_mixscape.py:198]: Why make this print so complex? Also, if this is the default behavior there's no need for the warning.

**Review by Zethson — CHANGES_REQUESTED** [review]

File: pertpy/tools/_mixscape.py (hunk 1 above)
Comment (xinyuejohn) [reply pertpy/tools/_mixscape.py:198]: Good point

File: pertpy/tools/_mixscape.py [hunk 2]
```diff
@@ -194,7 +194,7 @@ def mixscape(
         if layer is not None:
             X = adata_comp.layers[layer]
         else:
-            X = adata_comp.X
+            X = adata_comp.layers["X_pert"]
```
Comment (Zethson) [inline pertpy/tools/_mixscape.py:197]: We could wrap this with a try except and if the layer cannot be found we suggest to the user to calculate the perturbation signature first. Think this is what you wanted with your warning earlier?

Comment (xinyuejohn) [reply pertpy/tools/_mixscape.py:197]: exactly

Comment (Zethson) [reply pertpy/tools/_mixscape.py:197]: Could you have an attempt, please?
Use `print("[bold yellow] BLABLA")` for warnings

Comment (xinyuejohn) [reply pertpy/tools/_mixscape.py:197]: I used try. What do you think?

**Comment by Zethson** [comment]
Please add a `from rich import print` to the top and we're good!

**Pull request merged**